doc/python/dumbbell-plots.md

```diff
@@ -0,0 +1,176 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.8.0
+  plotly:
+    description: How to create dumbbell plots in Python with Plotly.
+    display_as: basic
+    language: python
+    layout: base
+    name: Dumbbell Plots
+    order: 19
+    page_type: example_index
+    permalink: python/dumbbell-plots/
+    thumbnail: thumbnail/dumbbell-plot.jpg
+---
+
+## Basic Dumbbell Plot
+
+
+Dumbbell plots are useful for demonstrating change between two sets of data points, for example, the population change for a selection of countries for two different years
```

Comment:
This is a cool page, thanks for adding it!

Some feedback:

1. Neither plot has a legend, because you're using a single trace with hardcoded colors, so there's no way of knowing what green and blue mean. I would suggest a separate trace per year and then another one for the line but hide the line from the legend.
2. Along similar lines, it might be cool to have a third example (or a modified second example) with some lines that go the other way and that are colored red, say. Again I would make this a different trace so it can have a legend entry.

This is not blocking feedback, just something that's on my mind as I've always wanted a PX function to do this stuff :)